template/build/webpack.base.conf.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
           postcss: cssUtils.postcss,
           loaders: merge({js: 'babel-loader'}, cssUtils.styleLoaders({
             sourceMap: useCssSourceMap,
-            extract: env.prod
+            extractCSS: env.prod
```

Comment:
Feel free to open a Github ticket if you feel something is wrong, or make a PR.
Thanks for taking time to fix the starter kit!